superset/connectors/druid/models.py

```diff
@@ -137,14 +134,14 @@ class DruidCluster(Model, AuditMixinNullable, ImportMixin):
     broker_user = Column(String(255))
     broker_pass = Column(EncryptedType(String(255), conf.get("SECRET_KEY")))
 
-    export_fields = (
+    export_fields = [
         "cluster_name",
         "broker_host",
         "broker_port",
         "broker_endpoint",
         "cache_timeout",
         "broker_user",
-    )
+    ]
```

Comment:
Is there some specific reason why these are made mutable? This is mostly out of scope for this PR, but I think in general it might be a good idea to have functions/methods expect `Sequence` instead of `List`/`Tuple` when the idea is just to iterate and read over the sequence of values and then leave it up to the caller to decide on whether or not to send across a `Tuple` or `List.

Reply:
@villebro I initially though these were defined in FAB but the base class is [here](https://github.com/apache/incubator-superset/blob/648f9fa54b90ce4cca84737a21214fd92c0e9fcc/superset/models/helpers.py#L55). `mypy` simply ensures that the types are consistent. 

Personally I like having differentiation between `List` and `Tuple` as it indicates whether the object is mutable or non-mutable.